Download/INSTALL

```diff
@@ -30,7 +30,7 @@ mostly by offering precompiled binaries are:
 
 Note, however, that these are updated independently and may not yet provide
 the latest GAP release. Further details are available on the GAP website
-here: <http://www.gap-system.org/Download/alternatives.html>.
+here: <https://www.gap-system.org/Download/alternatives.html>.
```

Comment:
This file is a copy of INSTALL.md from GAP. Maybe instead of copying it over to the GAP website, just put a link to it stable-4.8 version https://github.com/gap-system/gap/blob/stable-4.8/INSTALL.md ? I can make this in another PR.